esp32-rust/Dockerfile

```diff
@@ -3,14 +3,16 @@ FROM sergiogasquez/esp-rs-env:wokwi
 # Examples from esp-idf-hal
 RUN git clone https://github.com/esp-rs/esp-idf-hal.git rust-project
 
-RUN pip3 install esptool --user
-
-COPY compile.sh /home/esp
+# Copy compilation script
+COPY compile.sh /home/esp/
 
+ENV IDF_TOOLS_PATH=/home/esp/.espressif
 # Compile esp-idf-hal examples
 RUN cd rust-project \
- && export WOKWI_MCU="esp32-c3" \
- && /home/esp/compile.sh \
- && export WOKWI_MCU="esp32" \
- && /home/esp/compile.sh \
- && rm /home/esp/rust-project/build-out/project.bin
+    && export WOKWI_MCU="esp32" \
+    && bash /home/esp/compile.sh \
+    && export WOKWI_MCU="esp32-c3" \
+    && bash /home/esp/compile.sh \
+    && export WOKWI_MCU="esp32-s2" \
+    && bash /home/esp/compile.sh
```

Comment:
I think this line is missing a backslash at the end?

Reply:
True! I was also building `esp32s3` but then realized it does not make sense since it's not supported in Wokwi yet!